specificatie/domain.yaml

```diff
@@ -1106,6 +1106,12 @@ components:
           $ref: "#/components/schemas/ZakelijkRecht"
         persoon:
           $ref: "#/components/schemas/PersoonBeperkt"
+        indicatieGerechtigdUitSplitsing:
+          type: "boolean"
+          description: "Geeft aan of dit zakelijk gerechtigde is met een recht is dat onstaan is uit splitsing van een ander recht"
```

Comment:
Description klopt tekstueel niet. Teveel 'is' woorden

Reply:
Aangepast